client/layout/community-translator/launcher.jsx

```diff
@@ -16,23 +16,37 @@ import translator from 'lib/translator-jumpstart';
 import localStorageHelper from 'store';
 import Dialog from 'components/dialog';
 import analytics from 'lib/analytics';
+import userSettings from 'lib/user-settings';
 
 class TranslatorLauncher extends React.PureComponent {
 	static displayName = 'TranslatorLauncher';
 
 	static propTypes = {
-		isActive: PropTypes.bool.isRequired,
-		isEnabled: PropTypes.bool.isRequired,
+		translator: PropTypes.func,
```

Comment:
Oops. It should be `translate`, not `translator`. Now I'm wondering why `eslint` didn't catch it 🙃

Reply:
Fixed 👍